CONTRIBUTING.es.md

```diff
@@ -0,0 +1,176 @@
+Como comunidad le damos la bienvenida a cualquiera que desee escribir y enviar entradas al blog de Playful Programming. ¡Vamos! En este documento revisaremos los pasos necesarios para crear una nueva publicación, y después enviarla a PP como una *pull request* o solicitud para incorporar cambios.
+
+> [!NOTA]
```

Comment:
While this translation is correct, it will likely break GitHub's custom rendering for this block

```suggestion
> [!NOTE]
```